source/run-command/tests/unified/run-command.yml

```diff
@@ -0,0 +1,242 @@
+description: runCommand spec requirements
+
+schemaVersion: "1.5"
+
+createEntities:
```

Comment:
General comment for the YAML test here. I *think* we need to continue following the pattern established in the other tests in both aliasing and naming conventions. The first I *believe* is going to be needed for drivers that consume the YAML directly and do not use the generated JSON. (I would like confirmation here from those drivers - Ruby I think of first - @DmitryLukyanov can you confirm)? The second is just for general consistency - the other specs follow the "entityName0, entityName1" pattern and I think it's just better to stay with that for now rather than have a unified test that breaks the pattern. If that pattern suits us is another discussion.

Reply:
I think you meant to tag @comandeo for Ruby

Reply:
Put back the use of anchors, I would like to keep the human-ish names coll0/1 makes sense when you just need a collection but the three dbs I have here are declared differently to show the differences in behavior in each test, personally I find it easier to follow/write, but I don't have a strong attachment either.